.github/workflows/test.yml

```
@@ -0,0 +1,18 @@
+name: Backed Deployment
+
+on:
+  push:
+    branch: testing/Testing-scripts
```

I'm not sure about this, but since this will be merged with master, should the branch name be changed to master instead?

---

This is a matter of opinion, because right now there is now way to tell if we change it to `master` and this action work the same way as intended. It would probably work so changing this to `master` will not do any harm.

---

There is one more option though
```suggestion
    branch: [testing/Testing-scripts, master]
```